doc/release-process.md

```diff
@@ -93,6 +93,8 @@ Create the OS X SDK tarball, see the [OS X readme](README_osx.md) for details, a
 
 ### Optional: Seed the Gitian sources cache and offline git repositories
 
+NOTE: Gitian is sometimes unable to download files. If you have errors, make sure you have checked out the correct tag in bitcoin, then (re-)run the command(s) below.
```

Comment:
I'd merge these into one sentence "By default, Gitian will fetch source files as needed. To cache them ahead of time, make sure you have checked out the tag you want to build in bitcoin, then:"

This saves an extra bouncing-back step :)